ci/pyuvdata_min_versions_tests.yml

```diff
@@ -0,0 +1,26 @@
+name: pyuvdata_min_versions_tests
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - astropy==5.0.4
+  - astroquery==0.4.4
+  - h5py==3.1.*
+  - astropy-healpix==0.6
+  - numpy==1.19.*
+  - pyerfa==2.0
+  - python-casacore==3.3.*
+  - pyyaml==5.1.*
+  - scipy==1.3.*
+  - six  # added for now because sometimes an old python-casacore build (before 3.3.1) is used that omits this dependency
```

Comment:
so we tag astropy with a fixed final tick in the version number. Should we do the same with casacore to get rid of this extra baggage? Maybe it's not a big deal in the end, just seems like an easy solution to remove six when we don't need it.

Reply:
sure!